root/etc/e-smith/templates/etc/nethserver/otp.secret/10base

```diff
@@ -0,0 +1,25 @@
+{
+  use strict;
+  use warnings;
+  use esmith::ConfigDB;
+  use NethServer::Password;
```

Comment:
You don't need to open the ConfigDB: you already have access to it.
Also remove stricts and warnings which prevent normal template usage.

```suggestion
  use NethServer::Password;
```

Reply:
done